src/View.php

```diff
@@ -1061,6 +1061,9 @@ public function on($event, $selector = null, $action = null, $defaults = null)
             if ($ex instanceof self && $ex instanceof UserAction\JsExecutorInterface) {
                 //Executor may already had been add to layout. Like in CardDeck.
                 if (!isset($this->app->html->elements[$ex->short_name])) {
+                    // very dirty hack, @TODO, attach modals in the standard render tree
+                    // but only render the result to a different place/html DOM
+                    $ex->viewForUrl = $this;
```

Comment:
@ibelar I was able to finish this PR with this hack, but do you think we can/should add Modal like any other View to a standard render tree and then render like now?